.github/workflows/main.yml

```diff
@@ -32,6 +31,8 @@ jobs:
       - name: Release
         env:
           gh_ci_key: ${{ secrets.GH_CI_KEY }}
+          ziti_ci_gpg_key: ${{ secrets.ZITI_CI_GPG_KEY }}
```

Comment:
Was this snippet doc'ed anywhere?

Reply:
I have some doc I'm going to try and hand off to the security team